Misc/NEWS.d/next/Library/2025-03-11-23-58-45.gh-issue-131123.WB6tPh.rst

```diff
@@ -0,0 +1 @@
+Supported completions for attributes of convenience variables in :mod:`pdb`.
```

Comment:
Since it's a new feature, we also need a true What's New entry. Is there a need for improving the docs about what is a "convenience variable attribute"?

Reply:
I don't consider this as a new feature. Also I don't believe every single "new" behavior should make it to the whatsnew entry (at least that's not how we did it before). A significant behavior change or new feature that's available to user should be documented in whatsnew, but this is something that the user would just expect to work. The completion feature was there for a long time, even for the convenience variable itself. The attribute completion also works for normal variables. This is just a missing piece in our existing feature.

`convenience variable` is well documented in `pdb` docs, and `attribute` does not have any extra meanings outside of pure Python context - it's just attributes of the convenience variable (which is just an alias for a normal Python object).

Reply:
No worries. Yeah if the change is that "completions for attributes now work on all variables", then I agree it should be a whatsnew entry.